app/views/admin/dashboard/index.html.erb

```diff
@@ -1 +1,10 @@
 <h1> Hello, <%= current_user.name %>! Welcome to your Admin Dashboard!</h1>
+
```

Comment:
Maybe we could add a header here just describing the next section?

Reply:
good call